docs/README.md

```diff
@@ -1,6 +1,6 @@
-# Taurus Data Model Documentation
+# GEMD Documentation
 
-This is the next major data model developed by Citrine, codename `taurus`.
+GEMD stands for Generalized Expression of Materials Data (or Graph Encoding of Materials Data, if you prefer). It's an open source format initially developed by the fine folks at Citrine Informatics. 
```

Comment:
See above.

Reply:
I didn't actually remove anything substantive, I just moved things that are only relevant to contributors to contributing.md